api/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import com.google.protobuf.gradle.*
+
+plugins {
+    kotlin("jvm") version "1.7.21"
+    id("com.google.protobuf") version("0.9.1")
+    `java-library`
+    `maven-publish`
+}
+
+group = "io.chirpstack"
+version = "1.0-SNAPSHOT"
```

Comment:
Maybe should be `4.0.4-SNAPSHOT` and not `1.0-SNAPSHOT`.

Ref: https://github.com/chirpstack/chirpstack/commit/bfcff17f3414d7bc6063a97caa8a64c11cda9396

Reply:
I don't think we should set it to a particular version since people will probably forget to update the version number in these examples when modifying the protobuf definitions. Instead, wouldn't it be preferable to add a pipeline job that deploys a properly versioned artifact to maven central whenever a new version is tagged, or something along those lines?

Reply:
Maybe this could be integrated in https://github.com/chirpstack/chirpstack/blob/master/Makefile#L7? That tasks sets all the versions.

Reply:
Sure. Will do.

Reply:
I have set the version to "4.0.4" since I now know that the version is modified in the version target of the root Makefile.

I have also modified the version target to include updating the versions of the java and kotlin api clients.